compiler/native/transform.go

```diff
@@ -38,6 +38,16 @@ func (c *client) TransformStages(r *pipeline.RuleData, p *yaml.Build) (*pipeline
 	name := c.repo.GetName()
 	number := c.build.GetNumber()
 
+	// check if the compiler is setup for a local pipeline
+	if c.local {
+		// set a default for the org
+		org = "localOrg"
```

Comment:
these could be better suited as constants, but not a big deal

Reply:
I'll look to make that change in a future iteration 👍